pkg/ingestor/parser/cyclonedx/parser_cyclonedx_test.go

```diff
@@ -61,6 +61,20 @@ func Test_cyclonedxParser(t *testing.T) {
 		wantNodes: testdata.CycloneDXQuarkusNodes,
 		wantEdges: testdata.CyloneDXQuarkusEdges,
 		wantErr:   false,
+	}, {
+		name: "valid CycloneDX document where dependencies are missing dependsOn properties",
+		doc: &processor.Document{
+			Blob:   testdata.CycloneDXDependenciesMissingDependsOn.CycloneDXDependenciesMissingDependsOn,
```

Comment:
Looks like you accidentally added  `CycloneDXDependenciesMissingDependsOn`. This will fix the error.